scripts/ec2-stage

```diff
@@ -25,19 +25,10 @@ SOURCE_BUCKET=""
 KEY_NAME=""
 SECURITY_GROUP=""
 INSTANCE_PROFILE=""
-SUPPORTED_OSES=("linux" "windows")
 TARGET_OS_LIST=${SUPPORTED_OSES[@]}
```

Comment:
Since `publishing-common.sh` is sourced after `TARGET_OS_LIST=${SUPPORTED_OSES[@]}`, `TARGET_OS_LIST` ends up empty and this ends up doing nothing at line 178.